Source/CombatExtended.sln

```diff
@@ -74,6 +74,12 @@ Global
 		{6B518120-4315-4017-B4E6-7DA8ABCCFF84}.Debugger|Any CPU.Build.0 = Debug|Any CPU
 		{6B518120-4315-4017-B4E6-7DA8ABCCFF84}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{6B518120-4315-4017-B4E6-7DA8ABCCFF84}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4D883AAF-5689-4B17-AD63-DCB9E74684C8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
I think this project was the added sub-project that isn't needed.  Probably should nuke the changes to this file entirely.